pkg/server/config/config.go

```diff
@@ -118,4 +123,17 @@ func modifier(config *CycloneServerConfig) {
 		}
 	}
 
+	if config.SystemNamespace == "" {
```

Comment:
I'd prefer  to define a `Getter` to retrieve the namespace, and value set by `ENV` should have higher priority.